code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -611,11 +611,12 @@
 						return
 
 				if(!H.unacidable)
-					var/obj/item/organ/external/affecting = H.get_organ("head")
-					affecting.receive_damage(0, 75)
+					var/obj/item/organ/external/head/affecting = H.get_organ("head")
+					if(affecting)
+						affecting.receive_damage(0, 75)
+						affecting.disfigure()
```

Comment:
The above damage is always enough to disfigure, but on the offhand chance that we eventually have some species that's hyper-resistant to burn damage.....